### DISCUSSION TITLE
Bagaimana cara untuk membuat gRPC dengan menggunakan golang ?

### DISCUSSION THREAD

**Discussion by refs05** [community/community#55265] [Programming Help]
### Body

Bagaimana cara untuk membuat gRPC dengan menggunakan golang ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rmdhfz** [comment]
Berikut ini adalah referensi untuk membuat gRPC dengan golang:
https://dasarpemrogramangolang.novalagung.com/C-golang-grpc-protobuf.html
